cmd/tea_submitplan.go

```diff
@@ -680,6 +681,9 @@ func (m submitPlanModel) submitPlanCmd() tea.Msg {
 		m.processing <- submitPlanUpdateMsg{m.blastRadiusTask.UpdateStatusMsg(taskStatusError)}
 		m.processing <- submitPlanUpdateMsg{m.risksError("failed to update planned changes", err)}
 		close(m.processing)
+
+		// TODO: Ask David about the position of these too
+		sentry.CaptureException(err)
```

Comment:
I haven't dug in more closely what's happening here, but for all errors, there is https://github.com/overmindtech/cli/blob/6c2f3009462dfc73a19cea20377714c5a9b438c0/cmd/tea_terraform.go#L161, and `risksError` does return a `fatalError` that is propagated up, so it should already land in sentry.